src/passes/Print.cpp

```diff
@@ -78,6 +78,16 @@ struct PrintSExpression : public Visitor<PrintSExpression> {
     return name;
   }
 
+  std::ostream& printName(Name name) {
+    // we need to quote names if they have tricky chars
```

Comment:
Thanks, I didn't realize that. I guess this change makes binaryen accept more s-expressions than the spec, then. On the other hand it was useful for rust code and might be for others. Maybe I'll suggest changing this in the spec repo.